From: Christoph Goehre <chris@sigxcpu.org>
Date: Mon, 6 Jun 2011 20:41:24 +0200
Subject: downgrade SQLITE_VERSION to 3.7.1

To make squeeze backports easier.

Closes: #627598

---
 mozilla/configure.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/mozilla/configure.in b/mozilla/configure.in
index acfe1dd..ffca0e5 100644
--- a/mozilla/configure.in
+++ b/mozilla/configure.in
@@ -130,7 +130,7 @@ GCONF_VERSION=1.2.1
 GIO_VERSION=2.18
 STARTUP_NOTIFICATION_VERSION=0.8
 DBUS_VERSION=0.60
-SQLITE_VERSION=3.7.5
+SQLITE_VERSION=3.7.1
 LIBNOTIFY_VERSION=0.4
 
 MSMANIFEST_TOOL=
-- 
